test/tools/reporter/mongodb_reporter.js

```diff
@@ -135,10 +135,7 @@ class MongoDBMochaReporter extends mocha.reporters.Spec {
         timestamp = timestamp ? timestamp.toISOString().split('.')[0] : '';
 
         output.testSuites.push({
-          package:
-            suite.file.includes('functional') || suite.file.includes('integration')
-              ? 'Functional'
-              : 'Unit',
+          package: suite.file.includes('integration') ? 'Functional' : 'Unit',
```

Comment:
This is a bit of a throwaway field in the xunit output, it doesn't impact anything on EVG, should we just name it `integration` now?

Reply:
oh I have no idea what the output goes to... does the reporter only run on the integration directory now and isn't used for our actual unit tests, or do you just mean `Functional -> Integration`?

Reply:
The reporter works for both unit and integration cus its pointed at by the mocharc. And yea, I just mean the string "Integration". It's just a prop on the very root node of the xml output:

```xml
<testsuite package="Unit" id="0" name="..." timestamp="..." hostname=".." tests="0" failures="0" errors="0" time="0">
```